src/components/feilside/Feilside.less

```diff
@@ -1,64 +1,10 @@
-@coreModulePath: "./../../";
-@nodeModulesPath: "./../../";
-
-@import "../../../node_modules/nav-frontend-core/less/core";
-@import "../../../node_modules/nav-frontend-ekspanderbartpanel-style/src/index";
-@import "../../../node_modules/nav-frontend-chevron-style/src/index";
-@import "../../../node_modules/nav-frontend-chevron-style/src/mixins";
-@import "../../../node_modules/nav-frontend-spinner-style/src/spinner-style";
-@import "../../../node_modules/nav-frontend-alertstriper-style/src/index";
-
-// Størrelser:
-
-@fullBredde: 912px; // Maks bredde på et panel
```

Comment:
Dette kan jeg ikke godt nok. Men vil tro det fortsatt ser bra ut å lese feilsiden i ulike størrelser?

Reply:
Så vidt jeg har kunnet se, ser det fortsatt bra ut ;)  kan ikke  se at denne kodem ble brukt! Men rop gjerne ut (noen andre) hvis det er noe jeg har oversett!

Reply:
Om du ikke finner noen andre referanser til Feilside så har du nok helt rett. 
Det er mye død kode i prosjektene, kjempebra å få det ryddet opp :)

Reply:
Stylingen så fin ut sånn den var, men så nå at "FeilSide"-komponenten endte opp med å blande inn styling fra "SideIkkeFunnet".  så la til en comitt nå som renamer css-klassen for den siden, så de ikke mikses med hverandre. Ser det greit ut? :)